cmake/dependencies/AMReX.cmake

```diff
@@ -143,7 +143,12 @@ macro(find_amrex)
     else()
         message(STATUS "Searching for pre-installed AMReX ...")
         set(COMPONENT_PRECISION ${HiPACE_PRECISION} P${HiPACE_PRECISION})
-        find_package(AMReX 21.08 CONFIG REQUIRED COMPONENTS 3D ${COMPONENT_PRECISION} PARTICLES TINYP)
+        find_package(AMReX 21.08 CONFIG REQUIRED COMPONENTS 3D ${COMPONENT_PRECISION} PARTICLES)
```

Comment:
Btw, feel free to bump up this version when you update `HiPACE_amrex_branch`, too.

Reply:
Thanks! Done in #1033